router/dealer.go

```diff
@@ -692,43 +701,47 @@ func (d *dealer) cancel(caller *Session, msg *wamp.Cancel) {
 
 	// Cancel mode should be one of: "skip", "kill", "killnowait"
 	mode := wamp.OptionString(msg.Options, "mode")
```

Comment:
seems that we should validate that it is one of the expected modes, and default to "skip" if not set.